External/FEXCore/Source/Interface/Core/JIT/x86_64/JIT.cpp

```diff
@@ -648,7 +648,9 @@ void *X86JITCore::CompileCode(uint64_t Entry, [[maybe_unused]] FEXCore::IR::IRLi
     ThreadState->CTX->ClearCodeCache(ThreadState, false);
   }
 
+  ClearRelocations();
```

Comment:
Not specific to this commit, i think we should have a "cleanup after compile" to release this resources sooner.

Reply:
The ClearRelocations here is actually just to ensure sanitization with a `std::move` in a different part of the codebase. The vector should already be empty but std::move with std::vector is...weird. `Unless otherwise specified, such moved-from objects shall be placed in a valid but unspecified state.`

Reply:
I suppose I should probably just clear the relocation vector after move and remove this ClearRelocations.

Reply:
Added a vector clear At the end of each JIT's CompileCode function. Which will be exactly where the relocations vector gets moved.
Resolved.